### PR TITLE
rubygem-webpack-rails 0.9.8

### PR DESCRIPTION
**Pull request by dLobatog**
As https://github.com/theforeman/foreman-packaging/pull/1174/files#r67130109 needs it

### CODE REVIEW

**Comment by dLobatog** [comment]
Rebased

File: rubygem-webpack-rails/rubygem-webpack-rails.spec [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%doc %{gem_instdir}/MIT-LICENSE
+%{gem_instdir}/example
```
Comment (domcleal) [inline rubygem-webpack-rails/rubygem-webpack-rails.spec]: examples should be moved to the %files doc list instead

**Comment by dLobatog** [comment]
Updated to include examples in %doc

**Comment by domcleal** [comment]
Merged, thanks @dLobatog.

**Pull request merged**